webapp/src/index.tsx

```diff
@@ -47,10 +46,7 @@ export default class Plugin {
         registry.registerChannelHeaderButtonAction(<ServiceNowIcon className='servicenow-icon'/>, () => store.dispatch(toggleRHSPlugin), null, Constants.ChannelHeaderTooltipText);
         registry.registerAdminConsoleCustomSetting('ServiceNowUpdateSetDownload', DownloadButton);
 
-        // Store the Mattermost site URL in cookies
-        Cookies.set(Constants.SiteUrl, store.getState().entities.general.config.SiteURL as string);
-
-        const iconUrl = `${Utils.getBaseUrls().publicFilesUrl}${Constants.SERVICENOW_ICON_URL}`;
+        const iconUrl = `${Utils.getBaseUrls(store.getState().entities.general.config.SiteURL).publicFilesUrl}${Constants.SERVICENOW_ICON_URL}`;
```

Comment:
Probably good to avoid accessing `state.entities` etc. in this sort of context too.

```suggestion
        const iconUrl = `${Utils.getBaseUrls(getSiteURL(store.getState()).publicFilesUrl}${Constants.SERVICENOW_ICON_URL}`;
```